app/assets/javascripts/client/directives/learning_questions/learningQuestionModal.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('learningQuestionModal', learningQuestionModal);
+
+  function learningQuestionModal() {
+    return {
+      restrict: 'E',
+      scope: {
+        reminder: '@'
+      },
+      templateUrl: 'client/views/modals/learning_questions/add_learning_question.html',
+      link: function(scope) {
+        // The modal exists on the parent scope, so we have to reach up to it  to bind any closing actions on the modal.
+        var parentScope = scope.$parent;
```

Comment:
How safe and clean is this compared to emitting an event and capture it in the parent scope to dismiss the modal?